hw/ip/otbn/doc/_index.md

```diff
@@ -516,6 +516,9 @@ The data memory (DMEM) is 256b wide and read-write accessible from the base and
 When accessed from the base instruction subset through the `LW` or `SW` instructions, accesses must read or write 32b-aligned 32b words.
 When accessed from the big number instruction subset through the `BN.LID` or `BN.SID` instructions, accesses must read or write 256b-aligned 256b words.
 
+Both memories can be accessed through OTBN's register interface ({{< regref "DMEM" >}} and {{< regref "IMEM" >}}) only when OTBN is idle, as indicated by the {{< regref "STATUS.busy">}} flag.
```

Comment:
Are there any Danger Zones here? Can Ibex stall if the accesses are made while `STATUS.busy`?

Reply:
I guess also just documenting what the behavior is would be useful as well.

Reply:
Likely, but we haven't yet concluded the discussion on what should happen exactly. That's being tracked in #2696.